### PR TITLE
- fixes dotse/zonemaster-engine#300

### PR DESCRIPTION
**Pull request by vlevigneron**
Fixes dotse/zonemaster-engine#300

### CODE REVIEW

**Comment by sandoche2k** [comment]
@matsduf can you check

File: util/get_iana_address_spaces_infos.pl [hunk 1]
```diff
@@ -4,17 +4,114 @@
...
+
 my $iana_url = q{http://www.iana.org/assignments/};
-my $dest_dir = qq{$FindBin::Bin/../share/};
+my $dest_dir = qq{$FindBin::Bin/../share};
 my @files_details = (
     { name => q{iana-ipv4-special-registry.csv}, url => $iana_url.q{/iana-ipv4-special-registry/iana-ipv4-special-registry-1.csv}, ip_version => 4 },
     { name => q{iana-ipv6-special-registry.csv}, url => $iana_url.q{/iana-ipv6-special-registry/iana-ipv6-special-registry-1.csv}, ip_version => 6 },
 );
+my $backup_extension = q{.back};
+my $new_extension    = q{.new};
```
Comment (matsduf) [inline util/get_iana_address_spaces_infos.pl]: Why not use i.e. `mkstemp` from [File::Temp](http://perldoc.perl.org/File/Temp.html) for the temporary files?

**Comment by sandoche2k** [comment]
@vlevigneron if you agree with @matsduf review, pls. update

**Comment by sandoche2k** [comment]
getting no response from @vlevigneron, @matsduf do you think we could move this to 2018.2?

**Comment by matsduf** [comment]
Updated milestone to v2018.2 for issue #300 and this PR.

**Comment by vlevigneron** [comment]
Yes, I could use File::Temp module. Not a big deal to change.

I change that (amongst other things) and I'll update the PR afterwards.

le 17 avril, matsduf a ?crit :
> matsduf requested changes on this pull request.
> 
> Please give a few lines describing the changes.
> 
> >  my @files_details = (
>      { name => q{iana-ipv4-special-registry.csv}, url => $iana_url.q{/iana-ipv4-special-registry/iana-ipv4-special-registry-1.csv}, ip_version => 4 },
>      { name => q{iana-ipv6-special-registry.csv}, url => $iana_url.q{/iana-ipv6-special-registry/iana-ipv6-special-registry-1.csv}, ip_version => 6 },
>  );
> +my $backup_extension = q{.back};
> +my $new_extension    = q{.new};
> 
> Why not use i.e. `mkstemp` from [File::Temp](http://perldoc.perl.org/File/Temp.html) for the temporary files?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/dotse/zonemaster-engine/pull/387#pullrequestreview-112820909

-- 
	Vincent Levigneron  A.F.N.I.C.  Vincent.Levigneron@afnic.fr

**Comment by sandoche2k** [comment]
Working :-)

**Pull request merged**